src/clients/storage/StorageClientBase-inl.h

```diff
@@ -203,7 +203,7 @@ folly::Future<StatusOr<Response>> StorageClientBase<ClientType, ClientManagerTyp
         } else {
           auto partsId = getReqPartsId(request);
           invalidLeader(spaceId, partsId);
-          LOG(ERROR) << "Request to " << host << " failed: " << ex->what();
+          LOG(ERROR) << "Request to " << host << " failed.";
```

Comment:
Could the ex not be null sometimes in the else branch?

Reply:
Seems like when there was an error but not an apache::thrift::transport::TTransportException, it would run into this else branch. Not sure what would cause this though.